meeshkan/serve/mock/faker/schema_faker.py

```diff
@@ -0,0 +1,275 @@
+########################
+#### FAKER
+########################
+import json
+import random
+from functools import reduce
+from typing import Any, Mapping, Union, Sequence, cast
+
+import typing
+from faker import Faker
+from http_types import Request, Response
+from meeshkan.serve.mock.faker.faker_base import MeeshkanFakerBase
```

Comment:
I think we should stick to either relative imports or absolute imports but not mix them?